Color Picker/ColorPickerScreen.swift

```diff
@@ -127,6 +127,126 @@ private struct BarView: View {
 	}
 }
 
+struct ColorInputView: View {
+    @EnvironmentObject private var appState: AppState
+    @State private var textColor: Color = .primary
+    @Binding var inputColorText: String
+    @Binding var isTextFieldFocused: Bool
+    @Binding var isPreventingUpdate: Bool
+    let colorPanel: NSColorPanel
+    let textFieldFontSize: Double
+    let inputColorType: colorType
+    enum colorType: String, CaseIterable {
```

Comment:
This already exists: https://github.com/sindresorhus/System-Color-Picker/blob/2410aba60bc4d6f0194654f23865c485244e4761/Color%20Picker/Constants.swift#L72

Reply:
OK